spec/factories/hyrax_file_set.rb

```diff
@@ -24,7 +25,7 @@
           .assign_access_for(visibility: evaluator.visibility_setting)
       end
       file_set.file_ids = evaluator.files.map(&:id) if evaluator.files
-      file_set.original_file_id = evaluator.original_file.id if evaluator.original_file
+      file_set.original_file_id = evaluator&.original_file&.id || evaluator.files&.first&.id
```

Comment:
i'm not sure i'm following this. is it used somewhere i'm missing?

Reply:
Oh this might be left over from a previous try.